Shared/Extensions/TextBox/TextBoxBehavior.Xamarin.cs

```diff
@@ -5,8 +5,13 @@
 using System.Text;
 using System.Threading.Tasks;
 using Windows.System;
+#if WINUI
```

Comment:
this is xamarin.cs, should not have WINUI inside, add #if on top so that WINUI is not targeted